shapefil.h

```diff
@@ -27,22 +27,25 @@ extern "C"
 {
 #endif
 
-/************************************************************************/
-/*           Version related macros (added in 1.6.0)                    */
-/************************************************************************/
+    /************************************************************************/
+    /*           Version related macros (added in 1.6.0)                    */
+    /************************************************************************/
 
 #define SHAPELIB_VERSION_MAJOR 1
 #define SHAPELIB_VERSION_MINOR 6
 #define SHAPELIB_VERSION_MICRO 0
 
-#define SHAPELIB_MAKE_VERSION_NUMBER(major, minor, micro) \
-    ((major) * 10000 + (minor) * 100 + (micro))
+#define SHAPELIB_MAKE_VERSION_NUMBER(major, minor, micro)                      \
+    ((major)*10000 + (minor)*100 + (micro))
 
-#define SHAPELIB_VERSION_NUMBER \
-    SHAPELIB_MAKE_VERSION_NUMBER(SHAPELIB_VERSION_MAJOR, SHAPELIB_VERSION_MINOR, SHAPELIB_VERSION_MICRO)
+#define SHAPELIB_VERSION_NUMBER                                                \
+    SHAPELIB_MAKE_VERSION_NUMBER(SHAPELIB_VERSION_MAJOR,                       \
+                                 SHAPELIB_VERSION_MINOR,                       \
+                                 SHAPELIB_VERSION_MICRO)
 
-#define SHAPELIB_AT_LEAST(major, minor, micro) \
-    (SHAPELIB_VERSION_NUMBER >= SHAPELIB_MAKE_VERSION_NUMBER(major, minor, micro))
+#define SHAPELIB_AT_LEAST(major, minor, micro)                                 \
+    (SHAPELIB_VERSION_NUMBER >=                                                \
+     SHAPELIB_MAKE_VERSION_NUMBER(major, minor, micro))
```

Comment:
unrelated changes

Reply:
Yes, someone did not run the pre-commit hooks.